app/upgrades/test_util.go

```diff
@@ -0,0 +1,106 @@
+package upgrades
```

Comment:
let's rename this file to `staking_util_test.go`

Reply:
ok